settings/blt.settings.php

```diff
@@ -20,6 +20,12 @@
 $request_uri = getenv('REQUEST_URI');
 $http_x_request_id = getenv('HTTP_X_REQUEST_ID');
 
+// If trusted_reverse_proxy_ips is not defined, fail gracefully.
+$trusted_reverse_proxy_ips = isset($trusted_reverse_proxy_ips) ? $_trusted_reverse_proxy_ips : '';
```

Comment:
This probably should not have an underscore in `$_trusted_reverse_proxy_ips`

Reply:
How embarassing! Copy/paste fail. Thanks, Tim!